### PR TITLE
Fixed wrong content entry in db

### PR DESCRIPTION
**Pull request by ydementieiev**
Fix for [5843](https://adc.luxoft.com/jira/browse/FORDTCN-5841)

### Summary
Fixed wrong ccpu_version entry in modul_meta parameter in db  (policy_regular build)

This code was added by me in this [commit](https://github.com/LuxoftSDL/sdl_core/commit/309671f16072568421cf22f57d7747b78331f938)

### CODE REVIEW

File: src/components/policy/policy_regular/src/sql_pt_representation.cc [hunk 1]
```diff
@@ -476,10 +476,9 @@ void SQLPTRepresentation::GatherModuleMeta(
   LOG4CXX_INFO(logger_, "Gather Module Meta Info");
   utils::dbms::SQLQuery query(db());
   if (query.Prepare(sql_pt::kSelectModuleMeta) && query.Next()) {
-    *meta->ccpu_version = query.GetString(0);
```
Comment (AByzhynar) [inline src/components/policy/policy_regular/src/sql_pt_representation.cc:479]: @ydementieiev Why do you create PT with ccpu_version but removed possibility to get it from there?

Comment (ydementieiev) [reply src/components/policy/policy_regular/src/sql_pt_representation.cc:479]: @AByzhynar you are right, it was fall, thank you !!,  [added here](https://github.com/LuxoftSDL/sdl_core/pull/28/commits/62bc6f3dc160da972e4eecbcc36f172bb64a92be)

File: src/components/policy/policy_regular/src/policy_table/types.cc [hunk 2]
```diff
@@ -1354,7 +1354,6 @@ void ModuleMeta::ReportErrors(rpc::ValidationReport* report__) const {
 
 void ModuleMeta::SetPolicyTableType(PolicyTableType pt_type) {
   CompositeType::SetPolicyTableType(pt_type);
-  ccpu_version.SetPolicyTableType(pt_type);
```
Comment (AByzhynar) [inline src/components/policy/policy_regular/src/policy_table/types.cc:1357]: @ydementieiev are you sure that we dont need it here?

File: src/components/policy/policy_regular/src/sql_pt_representation.cc [hunk 3]
```diff
@@ -1414,10 +1414,9 @@ bool SQLPTRepresentation::SaveModuleMeta(const policy_table::ModuleMeta& meta) {
   }
   const int64_t odometer = *(meta.pt_exchanged_at_odometer_x);
 
-  query.Bind(0, *(meta.ccpu_version));
```
Comment (AByzhynar) [inline src/components/policy/policy_regular/src/sql_pt_representation.cc:1417]: @ydementieiev SaveModuleMeta  () does not need ccpu_version? Why?

Comment (ydementieiev) [reply src/components/policy/policy_regular/src/sql_pt_representation.cc:1417]: @AByzhynar we saved ccpu_version on SetMetaInfo() function, this line was added by me [here](https://github.com/LuxoftSDL/sdl_core/commit/309671f16072568421cf22f57d7747b78331f938), but they are not needed

File: src/components/policy/policy_regular/src/policy_table/types.cc (hunk 2 above)
Comment (ydementieiev) [reply src/components/policy/policy_regular/src/policy_table/types.cc:1357]: @AByzhynar  this line was added by me before [here](https://github.com/LuxoftSDL/sdl_core/commit/309671f16072568421cf22f57d7747b78331f938), but after testing we see that it was not needed

**Pull request merged**